src/utils/auth.js

```diff
@@ -0,0 +1,70 @@
+/* jshint node: true */
+/* jshint esversion: 6 */
+
+const getToken = (tenant) => {
+  const payload = { service: tenant, username: 'flowbroker' };
```

Comment:
flowbroker?

Reply:
No, it is incorrect. It should be something like backstage